code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -270,3 +270,178 @@
 		var/mob/living/silicon/robot/R = loc
 		if (R.cell)
 			R.cell.use(hitcost)
+
+/obj/item/weapon/melee/baton/harm
+	name = "harm baton"
+	desc = "A harm baton for incapacitating people with."
+	icon_state = "harmbaton"
+	item_state = "baton0"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	origin_tech = Tc_COMBAT + "=2;" + Tc_SYNDICATE + "=3"
+	attack_verb = list("robusts", "harms")
+	var/dial = 1
+
+/obj/item/weapon/melee/baton/harm/examine(mob/user)
+	..()
+	if(user.is_holding_item(src))
+		to_chat(user, "<span class='notice'>It has a small dial at the base.</span>") //if you add a feature and won't add a way to advertise it, no one is going to use it
+
+/obj/item/weapon/melee/baton/harm/attack_self(mob/user)
+	if(status && clumsy_check(user) && prob(50))
+		user.simple_message("<span class='warning'>You grab the [src] on the wrong side.</span>",
+			"<span class='danger'>The [name] blasts you with its power!</span>")
+		var/mob/living/L = user
+		L.apply_effect(AGONY, stunforce)
+		L.audible_scream()
+		playsound(loc, "sparks", 75, 1, -1)
+		deductcharge(hitcost)
+		return
+	if(bcell && bcell.charge >= hitcost)
+		status = !status
+		user.simple_message("<span class='notice'>[src] is now [status ? "on" : "off"].</span>",
+			"<span class='notice'>[src] is now [pick("drowsy","hungry","thirsty","bored","unhappy")].</span>")
+		playsound(loc, "sparks", 75, 1, -1)
+		if(status)
+			force += 10 + 2*(dial-1) //send someone into the shadow realm with 10
+			throwforce += 10 + 2*(dial-1) //it doesn't deal damage on throw, this is just a constistency thing, I guess
+			hitcost = 100 * (dial * dial) // 100 cost on 1, 10000 cost on 10, change power cell
+			stunforce = 10 * dial //welcome to the world of pain
+		else
+			force = 10
+			throwforce = 7
+		update_icon()
+	else
+		status = 0
+		force = 10
+		throwforce = 7
+		if(!bcell)
+			user.simple_message("<span class='warning'>[src] does not have a power source!</span>",
+				"<span class='warning'>[src] has no pulse and its soul has departed...</span>")
+		else if (bcell.maxcharge < hitcost)
+			to_chat(user, "<span class='warning'>[src] clicks but nothing happens. Something must be wrong with the battery.</span>")
+		else
+			user.simple_message("<span class='warning'>[src] is out of charge.</span>",
+				"<span class='warning'>[src] refuses to obey you.</span>")
+
+	add_fingerprint(user)
+
+/obj/item/weapon/melee/baton/harm/attack(mob/M, mob/user)
```

Comment:
>copying the entire proc

*scream

Reply:
eh, not much I can do, I placed ..() wherever I could to make thinner